frontend/src/components/form/UserForm.js

```diff
@@ -0,0 +1,80 @@
+import {useState} from 'react'
+import {FormMoverContext} from './FormMoverContext'
+import FormMover from './FormMover'
+import FormMoverControl from './FormMoverControl'
+import FormCreateEvent from './FormCreateEvent'
+import FormScheduleEvent from './FormScheduleEvent'
+import FormConfirm from './FormConfirm'
+import FormSuccess from './FormSuccess'
+import DataService from '../../services/dataService'
+
+const UserForm = () => {
+    
+  const [currentStep, setCurrentStep] = useState(1)
+
+    const [userData, setUserData] = useState('')
+    const [finalData, setFinalData] = useState([])
+  const steps = [
+      "Description",
+      "Schedule",
+      "Confirm",
+      "Success"
+    ];
+
+    const displayStep = (step) => {
+      switch(step){
+        case 1:
+          return <FormCreateEvent />
+        case 2:
+          return <FormScheduleEvent />
+        case 3:
+          return <FormConfirm />
+        case 4:
+         return <FormSuccess />
+         default:
+      }
+    }
+  const handleClick = (direction) => {
+    let newStep = currentStep
+    direction === "next" ? newStep++ : newStep--
+    newStep > 0 && newStep <= steps.length-1 && setCurrentStep(newStep)
+  }
+
+  const handleSubmit = async () => {
+    await DataService.create({data: userData})
+    setCurrentStep(4)
+  }
```

Comment:
I think it would be nice if we could add what data is being submitted by/from the frontend as a comment so that in the future if we ever need to refactor it, we don't have to start a treasure hunt.